README.md

```diff
@@ -135,7 +135,7 @@ var listener = {
   error: (err) => {
     console.error('The Stream gave me an error: ', err);
   },
-  completed: () => {
+  complete: () => {
```

Comment:
Oh, wait, is this README *already* the result of compiling those files? in that case, ignore...